setup.py

```diff
@@ -178,7 +178,12 @@ def detect_libb2(prefixes):
 
 
 with open('README.rst', 'r') as fd:
-    long_description = fd.read()
+    # remove badges
+    long_description = re.compile(r'^.. start-badges.*^.. end-badges', re.M | re.S).sub('', fd.read())
+    # remove |substitutions|
+    long_description = re.compile(r'\|\w+\|').sub('', long_description)
```

Comment:
*shrug*, okay. my priority was to make this work in the long term, regardless of how it *may* damage the actual readability, because the worst case of going back to cleartext is worse...